CONTRIBUTING.md

```diff
@@ -0,0 +1,180 @@
+Contributing to the ResearchKit Framework
+===========================
+
+This page focuses on code contributions to the existing
+codebase. However, other types of contributions are welcome too, in
+keeping with the ResearchKit™ framework [best practices](best_practices). For example,
```

Comment:
Since this file has been moved, the link should point to:
(../../wiki/best_practices)